adapters/spi/tomcat-adapter-spi/src/main/java/org/keycloak/adapters/tomcat/PrincipleFactory.java

```diff
@@ -0,0 +1,11 @@
+package org.keycloak.adapters.tomcat;
+
+import org.apache.catalina.Realm;
+import org.apache.catalina.realm.GenericPrincipal;
+
+import java.security.Principal;
+import java.util.Set;
+
+public interface PrincipleFactory {
```

Comment:
This is misspelled, please rename to `PrincipalFactory`

Reply:
fixed this and rebased onto latest master